api/test/unit/utils/attribution_test.py

```diff
@@ -0,0 +1,40 @@
+import re
+
+import pytest
+
+from catalog.api.utils.attribution import get_attribution_text
+
+
+@pytest.mark.parametrize(
+    "args, expectation",
+    [
+        (
+            ("Title", "Creator", "xy", "0.0", "https://license/url"),
+            '"Title" by Creator is licensed under CC XY 0.0. '
+            "To view a copy of this license, visit https://license/url.",
+        ),
+        (
+            ("", "Creator", "xy", "0.0", "https://license/url"),  # No title
```

Comment:
Is it possible any of these values could be `None`? Is it worth testing for those cases if so?

Reply:
Everything except `license` can be null; `license` is a non-nullable field in the model so we can assume it will always be present. Added the corresponding test case in 7f080f1.